ingest/sqs/service/sqs_streaming_service_test.go

```diff
@@ -44,193 +45,105 @@ func TestSQSServiceTestSuite(t *testing.T) {
 	suite.Run(t, new(SQSServiceTestSuite))
 }
 
-// This is a sanity-check test ensuring that all pools are tracked and returned correctly.
-func (s *SQSServiceTestSuite) TestProcessBlock() {
+// This test validates that the service can recover from an error or panic
```

Comment:
Note to reviewer: a lot of the logic that was tested here previously is now moved to the relevant block update processors